lib/states/tx/houston.R

```diff
@@ -3,82 +3,147 @@ source("common.R")
 
 # VALIDATION: [YELLOW] The Houston PD's Annual Reports don't list traffic
 # figures, but the stop counts don't appear unreasonable for a city of 2M
-# people. 2018 only has partial data.
+# people. 2018 only has partial data. 
+# NEW DATA UPDATE: Number of stops on 2019 is roughly on par with number
+# of stops in 2017. New data for 2018 is the same as old 2018 data (need to fix). 
 load_raw <- function(raw_data_dir, n_max) {
-  d <- load_years(raw_data_dir, n_max = n_max)
-  bundle_raw(d$data, d$loading_problems)
+  
+  # old data 
+  old_2014 <- load_single_file(raw_data_dir, '2014.csv', n_max = n_max)
+  old_2015 <- load_single_file(raw_data_dir, '2015.csv', n_max = n_max)
+  old_2016 <- load_single_file(raw_data_dir, '2016.csv', n_max = n_max)
+  old_2017 <- load_single_file(raw_data_dir, '2017.csv', n_max = n_max)
+  
+  # new data: note that old data only had part of 2018, so we use new 
+  # data for 2018, which has the full year
+  new_2018 <- load_single_file(raw_data_dir, '2018.csv', n_max = n_max) 
+  new_2019 <- load_single_file(raw_data_dir, '2019.csv', n_max = n_max)
+  new_2020 <- load_single_file(raw_data_dir, '2020.csv', n_max = n_max)
+  
+  old_d <- bind_rows(
+    old_2014$data,
+    old_2015$data,
+    old_2016$data,
+    old_2017$data
+  )
+  
+  new_d <- bind_rows(
+    new_2018$data,
+    new_2019$data,
+    new_2020$data %>% rename("DISPOTION" = "DISPOSITION_NAME")
+  ) %>%
+    rename(`Case Number` = Case_Nbr,
+           `Offense Date` = Offense_Dtm,
+           Block = VIO_STREET_NBR,
+           Street = VIO_STREET,
+           `Scnd Block` = Scnd_Blk_Mile,
+           `Scnd Street` = Scnd_St_Name,
+           Race = RACE,
+           Gender = GENDER,
+           `Defendant Name` = DfndtName,
+           `Citataion Num` = CITATION_NUMBER,
+           Speed = Driving_Speed,
+           `Posted Speed` = Posted_Speed,
+           `V Color` = Veh_Color,
+           `V Make` = Veh_Make,
+           `V Model` = Veh_Model,
+           `Violation Description` = VioDesc,
+           `Officer Name` = OFFICER_NAME,
+    ) %>%
+    select(-VioCode, -Prim_Mile_Post_Number, -Scnd_Mile_Post_Number,
+           -CaseStatus, -DISPOSITION, -DISPOTION)
```

Comment:
Same as above—the `rename`ing and `coalesce`ing should be moved into `clean` instead of `load_raw`.